src/main/scala/org/holmesprocessing/totem/services/dnsmeta/service.conf.example

```diff
@@ -1,6 +1,10 @@
-[settings]
-Port        = 8080
+{
```

Comment:
some of the configurations use the format 
```
{
    "settings": ...
    <service_name>: ...
}
```

while others just go straight into the configuration options. We should make this uniformed